### PR TITLE
Make SpeechSynthesisUtterance events not bubble

### PR DESCRIPTION
**Pull request by foolip**
Closes https://github.com/WICG/speech-api/issues/103.

The following tasks have been completed:

 * [x] Updated web-platform-tests: https://github.com/web-platform-tests/wpt/pull/34710

Implementation commitment:

 * [x] Blink: Chrome passes added test (when run manually)
 * [x] Gecko: Firefox passes added test (when run manually)
 * [x] WebKit: STP 148 (but not Safari 15.5) passes added test (when run manually)

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/speech-api/pull/104.html" title="Last updated on Jul 6, 2022, 8:16 AM UTC (0375d18)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/speech-api/104/a099053...0375d18.html" title="Last updated on Jul 6, 2022, 8:16 AM UTC (0375d18)">Diff</a>

### CODE REVIEW

**Comment by foolip** [comment]
@marcoscaceres there isn't anyone who can review this, somehow I'm the only "editor" (in name only) with write access. Do you feel like making yourself an "editor" too?

**Comment by marcoscaceres** [comment]
hehe, sure :)

**Comment by marcoscaceres** [comment]
On a more serious note... I guess I should keep an eye on this as it is supported in WebKit.

**Comment by foolip** [comment]
@marcoscaceres cool, I'll wait for you to swoop in with a green review checkmark then :)

**Review by marcoscaceres — APPROVED** [review]

**Comment by marcoscaceres** [comment]
(if you have admin rights, you might need to add me as a collaborator to the repo... or else it's that weird grey tick)

**Comment by foolip** [comment]
I don't have admin rights, I was hoping you had someone close at hand with WICG admin rights.

**Comment by marcoscaceres** [comment]
Pinged @yoavweiss on email and cc'ed you.

**Comment by yoavweiss** [comment]
@marcoscaceres gave you write access. Lemme know if that's sufficient.

**Comment by marcoscaceres** [comment]
Thanks @yoavweiss. Should be fine, I think.

**Pull request merged**
